import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -0,0 +1,454 @@
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package integration
+
+import (
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/adminservices"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
+	"testing"
+)
+
+const (
+	applicationThrottlePolicyFlag  = "app"
+	subscriptionThrottlePolicyFlag = "sub"
+	advancedThrottlePolicyFlag     = "advanced"
+	customThrottlePolicyFlag       = "custom"
+)
+
+// Export an Application Throttling Policy from one environment and import to another environment
+func TestExportImportApplicationThrottlePolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, dev, user.Admin.Username, user.Admin.Password, apim.ApplicationThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     applicationThrottlePolicyFlag,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
+			adminPassword := adminservices.AdminPassword
+			if isTenantUser(args.CtlUser.Username, TENANT1) {
+				adminUsername = adminUsername + "@" + TENANT1
+			}
+			testutils.ValidateThrottlePolicyExportImport(t, args, adminUsername, adminPassword, apim.ApplicationThrottlePolicyType)
+		})
+	}
+}
+
+// Export a Subscription Throttling Policy from one environment and import to another environment
+func TestExportImportSubscriptionThrottlePolicy(t *testing.T) {
+
+	for _, user := range testCaseUsers {
+		t.Run(user.Description, func(t *testing.T) {
+			dev := GetDevClient()
+			prod := GetProdClient()
+
+			newPolicy := testutils.AddNewThrottlePolicy(t, dev, user.Admin.Username, user.Admin.Password, apim.SubscriptionThrottlePolicyType)
+			throttlePolicy, _ := testutils.PolicyStructToMap(newPolicy)
+			args := &testutils.PolicyImportExportTestArgs{
+				CtlUser:  testutils.Credentials{Username: user.CtlUser.Username, Password: user.CtlUser.Password},
+				Policy:   throttlePolicy,
+				Type:     subscriptionThrottlePolicyFlag,
+				SrcAPIM:  dev,
+				DestAPIM: prod,
+				Update:   false,
+			}
+			adminUsername := adminservices.AdminUsername
```

Comment:
shall we use superAdminUser and superAdminPassword variables directly without using adminservices.AdminUsername and adminservices.AdminPassword.

Refer [1].

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/api_test.go#L84

Reply:
fixed with 479b4168267adb70bde906548a563dade6a07df8